templates/universal-standard/script/universalresults.hbs

```diff
@@ -82,6 +82,9 @@ ANSWERS.addComponent("UniversalResults", Object.assign({}, {
     mapConfig: Object.assign({
       apiKey: HitchhikerJS.getDefaultMapApiKey("{{ mapConfig.mapProvider }}")
     },
+    {"providerOptions": {
```

Comment:
nit: new line before/after the top level curly braces

Reply:
updated!